code/modules/hydroponics/grown/kudzu.dm

```diff
@@ -52,7 +52,7 @@
 	var/text_string = ""
 	for(var/datum/spacevine_mutation/SM in mutations)
 		text_string += "[(text_string == "") ? "" : ", "][SM.name]"
-	text += "\n- Plant Mutations: [(text_string == "") ? "None" : text_string]"
+		text += "Plant Mutations: [(text_string == "") ? "None" : text_string]"
```

Comment:
Tab whoopsie in here since this tab makes it so the mutations will be printed out repeatedly.